packages/mobile/src/components/details-tile/DetailsTile.tsx

```diff
@@ -266,10 +268,12 @@ export const DetailsTile = ({
 
     const dogEarType = showPremiumDogEar
       ? isOwner
-        ? premiumConditions.nft_collection
-          ? DogEarType.COLLECTIBLE_GATED
-          : DogEarType.SPECIAL_ACCESS
-        : DogEarType.LOCKED
+        ? isUSDCEnabled && premiumConditions.usdc_purchase
```

Comment:
Was this supposed to set the shopping cart icon? It appears to be following through to the existing ternary if usdc is enabled and the premium condition has usdc?
Or am I misreading it?

Reply:
whoops that's why we need the helper fn lol